tests/doubleBuffer.xlmap

```diff
@@ -2,12 +2,87 @@
 
   <redirectedRegister name="doubleBuffer">
     <targetDevice><par>target</par></targetDevice>
-    <targetRegister>APP.0.DAQ0_BUF0</targetRegister>
+    <targetRegister>/APP/0/DAQ0_BUF0</targetRegister>
```

Comment:
I don't know from where Tomasz got this and if it was somehow realistic. Of course we can change it

Reply:
Btw the commit above shows that I was trying out different Path syntax styles, since I suspected a problem there, but there I was being mislead by the debugger